### ISSUE TITLE
New bugs in command 

### ISSUE THREAD

**Issue by MrLivixx** [Sank6/Discord-Bot-List#55]
I found a bug in the ``bots`` commands and ``list``
<img src="https://i.imgur.com/5I5WWaT.png">

And the "queue" command doesn't work yet`` there is a bot on check and in the list there is no
<img src="https://i.imgur.com/Zg3L2W4.png">

**Comment by inhydrox** [comment]
The bot uses master version of discord.js so instead of using
```js
client.users.get("some-id")
``` 
uses 
```js
client.users.cache.get("some-id")
```

**Comment by inhydrox** [comment]
I'm not sure about the second error

**Comment by MrLivixx** [comment]
Everything worked in the previous version

**Comment by bobby365** [comment]
I'm getting that same problem

**Comment by Sank6** [comment]
Fixed: a7dc229
